proving_system/stark/src/cairo_vm/cairo_mem.rs

```diff
@@ -1,22 +1,26 @@
 use super::errors::CairoImportError;
-use lambdaworks_math::{traits::ByteConversion, unsigned_integer::element::U256};
-use std::fs;
+use crate::FE;
+use lambdaworks_math::traits::ByteConversion;
+use std::{collections::HashMap, fs};
 
-#[derive(Clone, Debug, PartialEq)]
-pub struct CairoMemoryCell {
-    pub address: u64,
-    // This could also be a StarkField
-    // Since we need to convert everything to StarkFields
-    // later, we won't do it here
-    pub value: U256,
-}
+// `FE` is used as the type of values stored in
+// the Cairo memory. We should decide if this is
+// correct or we should consider another type.
 #[derive(Clone, Debug, PartialEq)]
 pub struct CairoMemory {
-    pub cells: Vec<CairoMemoryCell>,
+    data: HashMap<u64, FE>,
 }
 
 impl CairoMemory {
-    fn from_bytes_le(bytes: &[u8]) -> Result<Self, CairoImportError> {
+    pub fn new(data: HashMap<u64, FE>) -> Self {
+        Self { data }
+    }
+
+    pub fn get(&self, addr: &u64) -> Option<&FE> {
+        self.data.get(addr)
+    }
```

Comment:
It's not clear what a get function without description does. It could be renamed or at least some documentation could be added